x-pack/plugins/endpoint/public/applications/endpoint/mocks/app_context_render.tsx

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
New Mock utility that:
- initializes instances of `store`, `history`, `coreStart` (mocked), `depsStart` (from existing mock)
- provide a `AppWrapper` component that uses the above instances to initialize an App Root Context and inserts `children` to it when rendered.
- A custom `@testing-library/react` `render()` function that will use the provided `AppWrapper` to render the provided UI.

This should cut down on the amount of setup duplication across our UT.